code_challenges.rb

```diff
@@ -7,8 +7,11 @@
 # Expected output: [-9, 7, 9, 199]
 full_arr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
+fake_arr2 = [6, 8, 1, -6, 42, 2]
+# Expected Fake output: [-6, 1, 2, 6, 8, 42]
```

Comment:
so this one isn't complete but I dont see any comments or anything? how did you attempt to complete this one and what issues did you run into? :)